aws-node-termination-handler.yaml

```
@@ -0,0 +1,54 @@
+package:
+  name: aws-node-termination-handler
+  version: "1.25.1"
+  epoch: 0
+  description: Gracefully handle EC2 instance shutdown within Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 5641359c91288d0589c3c188309ce2771eb93838
+      repository: https://github.com/aws/aws-node-termination-handler
+      tag: v${{package.version}}
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/oauth2@v0.27.0
+        golang.org/x/net@v0.38.0
+
+  - uses: go/build
+    with:
+      ldflags: -X github.com/google/ko/pkg/commands.Version=${{package.version}}
+      output: node-termination-handler
+      packages: ./cmd/node-termination-handler.go
+      tags: nthlinux
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compatibility package for the ${{package.name}}
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/
+          ln -sf /usr/bin/node-termination-handler ${{targets.contextdir}}/node-termination-handler
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+      pipeline:
+        - runs: test "$(readlink /node-termination-handler)" = "/usr/bin/node-termination-handler"
+
+update:
+  enabled: true
+  github:
+    identifier: aws/aws-node-termination-handler
+    strip-prefix: v
+
+test:
+  pipeline:
+    - runs: |
+        node-termination-handler -h
```

```suggestion
        set -o pipefail
        node-termination-handler -h | grep -F -e "some output from -h"
```
Lets just make sure this smoke test doesnt blow up somehow. Also, any chance we could do `--version` or some equivalent?

---

nope there is no version subcmd or flag.

---

but its failing in grep somehow, so I leave it like this.